spec/models/measurement_unit_spec.rb

```diff
@@ -45,10 +45,11 @@
     end
 
     context 'with missing measurement unit present in database' do
+      subject! { MeasurementUnit.measurement_unit(unit_code) }
+
       before { allow(Raven).to receive(:capture_message).and_return(true) }
-      before { allow(MeasurementUnit).to receive(:measurement_units).and_return({}) }
 
-      subject! { MeasurementUnit.measurement_unit(unit_code) }
```

Comment:
This looks wrong - previously the sequence was

1. First `before` - watch `Raven`
2. Second `before`
3. `subject!` - which will perform an action which could potentially trigger `Raven`

Now the sequence is

1. `subject!` - perform action which may trigger `Raven`
2. first `before` - set up watching of `Raven` - this is now too late
3. second `before`